visualizations/crossbio_blast/config/crossbio_blast.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE visualization SYSTEM "../../visualization.dtd">
+<visualization name="BLAST XML Viewer">
+    <data_sources>
+        <data_source>
+            <model_class>HistoryDatasetAssociation</model_class>
+            <test type="isinstance" test_attr="datatype" result_type="datatype">blast.BlastXml</test>
```

Comment:
@peterjc can you confirm that this is not working for you? If you change it to `xml.GenericXml` or `data.Text`it should work.
Another reason to move the BLAST datatypes into the core?